src/index.js

```diff
@@ -0,0 +1,498 @@
+const inputTime = document.getElementById('timeInput');
+let dateSpan = document.querySelector('.dateSpan');
+let timeSpan = document.querySelector('.timeSpan');
+
+inputDate.onchange = changeDateAndTime;
+inputTime.onchange = changeDateAndTime;
+function changeDateAndTime () {
+  dateSpan.innerHTML = inputDate.value
+  timeSpan.innerHTML = inputTime.value
+}
+const selectTicket = document.querySelector('.inputItems__value__section')
+
+let options = document.querySelectorAll('.optionTicket')
+// console.log(options[0])
+let valueTicketsSpan = document.querySelector('.valueSpan')
+    selectTicket.onchange = testRadio;
+
+
+const email = document.getElementById('email')
+const error = document.querySelector('.error')
+
+email.onblur = function() {
+  if (!email.value.includes('@')) { 
+    email.classList.add('invalid');
+    error.innerHTML = 'Пожалуйста, введите правильный email.'
+  }
+};
+
+email.onfocus = function() {
+  if (this.classList.contains('invalid')) {
+    this.classList.remove('invalid');
+    error.innerHTML = "";
+  }
+};
+
+const name = document.getElementById('name')
+const errorForName = document.getElementById('errorForName')
+
+name.onblur = function() {
+  if (name.value.includes('!','@','#','$','%','^','&','*','(',')','-','_','+','=','~')) { 
+    name.classList.add('invalidForName');
+    errorForName.innerHTML = 'Имя введено не коректно.'
+  }
+};
+
+name.onfocus = function() {
+  if (this.classList.contains('invalidForName')) {
+    this.classList.remove('invalidForName');
+    errorForName.innerHTML = "";
+  }
+};
+console.log('testTrigger')
+
+let map = L.map('map').setView([53.891648, 25.303462], 20);
+L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png', {
+    maxZoom: 19,
+    attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+}).addTo(map);
+
+let marker = L.marker([53.891648, 25.303462]).addTo(map);
+let marker2 = L.marker([53.891741, 25.303519]).addTo(map);
+let marker3 = L.marker([53.891633967815714, 25.304059931370258]).addTo(map);
+let marker4 = L.marker([53.89153746078612, 25.303522330963546]).addTo(map);
+let marker5 = L.marker([53.89166321232616, 25.30312285250748]).addTo(map);
+
+
+const moveto = new MoveTo();
+const scrolls = document.querySelectorAll('.scroll');
+scrolls.forEach(element => element.addEventListener('click',handleClick))
+
+function handleClick(event) {
+  event.preventDefault()  
+  console.log(this.href)
+  const target = this.getAttribute('href');
+  moveto.move(document.querySelector(target))
+}
+
+const burger = document.querySelector('.burger');
+burger.addEventListener('click', burgerChange)
+
+  function burgerChange () {
+    if(screen.width > 768){
+  burger.classList.toggle('burgerX')
+  document.querySelector('.Welcome__inf').classList.toggle('newNav')
+  document.querySelector('.Menu-ForBurger').classList.toggle('Menu-ForBurger-Open')
+    }
+    else if(screen.width <= 768){
```

Comment:
Тут будет достаточно только else { 

} без if (screen.width <= 768).

Потому что у тебя в твоем первом ифе уже идет проверка что ширина БОЛЬШЕ 768. Тогда else выполнится когда ширина будет МЕНЬШЕ ЛИБО РАВНА 768.

```suggestion
    } else {
```